HyperVExtension/src/HyperVExtension/Providers/HyperVProvider.cs

```diff
@@ -69,10 +75,21 @@ public IAsyncOperation<ComputeSystemsResult> GetComputeSystemsAsync(IDeveloperId
         {
             try
             {
+                if (!string.IsNullOrEmpty(_hyperVPreReqErrorText))
+                {
+                    // Dev Home contains code and error notifications to add the user to the admin group and enable Hyper-V.
+                    // So we do not need to send this error back to Dev Home as it will result in duplication.
+                    return new ComputeSystemsResult(new List<IComputeSystem>());
+                }
+
                 var computeSystems = _hyperVManager.GetAllVirtualMachines();
                 _log.Information($"Successfully retrieved all virtual machines on: {DateTime.Now}");
                 return new ComputeSystemsResult(computeSystems);
             }
+            catch (VirtualMachineManagementServiceException serviceException)
+            {
+                return new ComputeSystemsResult(serviceException, _stringResource.GetLocalized("HyperVVirtualMachineManagementServiceError"), serviceException.Message);

```

Comment:
Should we log this?

Reply:
I actually log the original error in the `StartVirtualMachineManagementService()`  methond in the `HyperVManager` file